src/main/webapp/feedback/parsed-icon-feedback.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import IconFeedbackUtil from './icon-feedback-util.js';
+
+/** Stores parsed icon feedback data. */
+export default class ParsedIconFeedback {
+  /* Values of each icon feedback type at each interval. */
+  #counts = {
```

Comment:
this seems like duplicate code to makeCountDictionary. Can you merge the logic together.

Reply:
`counts` is a dictionary mapping `type` -> `array of counts`
`makeCountDictionary` returns a dictionary mapping `type` -> `count`

After parsing each interval, each value in the return of `makeCountDictionary` is appended to the value of the respective `type` in `counts`

Reply:
Please see my previous comment about making this into a class.

Reply:
Fixed